integrations/airflow/marquez_airflow/utils.py

```diff
@@ -19,7 +19,13 @@
 from airflow.models import Connection
 from airflow.utils.db import provide_session
 from airflow.version import version as AIRFLOW_VERSION
-from pendulum import Pendulum
+
+try:
+    # Import from pendulum 1.x version
+    from pendulum import Pendulum
+except:
```

Comment:
Looks like using a bare **except** fails the CI build [#7010](https://app.circleci.com/pipelines/github/MarquezProject/marquez/839/workflows/8964ba0d-9b54-4384-a673-50dee5f8b1dd/jobs/7010/parallel-runs/0/steps/0-103). Mind specifying the exception? (see Flake8 [E722](https://www.flake8rules.com/rules/E722.html)).